data/ua-update.json

```diff
@@ -65,5 +65,6 @@
   "google.at": "Linux#Maemo",
   "twitter.com": "Linux#Android 8.1.0",
   "apkpure.com": "Linux#Android 8.1.0",
-  "zingmp3.vn": "Linux#Android 8.1.0"
+  "zingmp3.vn": "Linux#Android 8.1.0",
+  "forbes.com": "Linux#Android 8.1.0"
```

Comment:
Btw, would dropping Linux be enough?

Like:
```
"forbes.com": "Linux# "
```

Space after the hash

Reply:
Nope, needs Android. Works without that "8.1.0" though.

Reply:
Others have that 8.1.0 => so we can be it with this as well.